samples/secret-santa/README.md

```diff
@@ -0,0 +1,57 @@
+---
+page_type: sample
+languages:
+- python
+author: delbert murphy
+ms.author: delbertm@microsoft.com
+products:
+- azure-qio
+- azure-quantum
+description: "Learn how to solve a binary optimization problem using the Azure Quantum optimization service"
+urlFragment: azure-quantum.secret-santa
+---
+
+# Solving the secret santa scenario with the Azure Quantum optimization service
```

Comment:
Santa should be capitalised, would suggest just capitalising Secret Santa everywhere for simplicity

```suggestion
# Solving the Secret Santa scenario with the Azure Quantum optimization service
```